tutorials/schema_dataframes.ipynb

```diff
@@ -176,7 +176,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "`IterableDataFrame` is very special, because it's not bounded. But one important feature of Fugue `IterableDataFrame` is that, it is empty aware, so at any point you can check if the dataframe is empty and peek the current first row, it will not affect the iteration. If you are interested in details, read [this](https://triad.readthedocs.io/en/latest/api/triad.utils.html#triad.utils.iter.EmptyAwareIterable)"
+    "`IterableDataFrame` is incredibly special because it's not bounded. But one important feature of Fugue `IterableDataFrame` is that it is empty aware, so at any point you can check if the dataframe is empty and peek the current first row, it will not affect the iteration. If you are interested in the detail, read [this](https://triad.readthedocs.io/en/latest/api/triad.utils.html#triad.utils.iter.EmptyAwareIterable)"
```

Comment:
this should probably be details? maybe we take out incredibly? sounds a bit dramatic.